### PR TITLE
feat: added ArrowControl component

### PR DESCRIPTION
**Pull request by klubiniecki-qantas**
## Description

Added ArrowControl component for image gallery controls.

💁‍

## Looks like this

Screen Shot 2021-10-01 at 1 37 03 pm

🤨

---

**PR check-list**

🚨 **IMPORTANT:** Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository first! 👀

- [x] You've requested a review from at least ONE person. Preferably a roo-ui maintainer ([Philip](https://github.com/philipwindeyer), [Eric](https://github.com/eneo5541), [Mike](https://github.com/memcc), [Marcel](https://github.com/Anthropic), [Sam](https://github.com/SamuSan) [Long](https://github.com/KieraDOG)) or a Roo UI working-group human.
  - Note: If your PR has not been reviewed within two days of requesting a review, feel free to seek reviews from other [team members of Qantas Hotels](https://github.com/orgs/hooroo/people).
- [x] Your final merge commit message is in the format `<type>(<scope>): <description>` as per [the contributing guide](https://github.com/hooroo/roo-ui/blob/master/.github/CONTRIBUTING.md#commit-naming)
  - Example: `fix(dependencies): minor version bump to resolve security vulnerability in set-value`

### CODE REVIEW

**Comment by ehyland** [comment]
👷 Deploy Preview for *roo-ui* processing.


🔨 Explore the source changes: b77eaaa784a6eab5f7e5b5ebab6e8fae377068cb

🔍 Inspect the deploy log: [https://app.netlify.com/sites/roo-ui/deploys/61568292d918230008674e7c](https://app.netlify.com/sites/roo-ui/deploys/61568292d918230008674e7c?utm_source=github&utm_campaign=bot_dl)

File: src/components/ArrowControl/ArrowControl.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { themeGet } from 'styled-system';
+import { rem } from 'polished';
+import { Icon, NakedButton } from '..';
+
+const ArrowControl = ({ direction, colorMode, onClick }) => {
+  const Button = styled(NakedButton)`
+    position: relative;
+    border-radius: 50%;
```
Comment (apalski) [inline src/components/ArrowControl/ArrowControl.js:11]: suggestion: to use the theme value for this:
`border-radius: ${themeGet('radii.rounded')};`

File: src/components/ArrowControl/ArrowControl.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      opacity: 1;
+    }
+  `;
+
+  const StyledIcon = styled(Icon)`
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  `;
```
Comment (KieraDOG) [inline src/components/ArrowControl/ArrowControl.js:28]: We should always declare styled component outside of component

**Review by KieraDOG — CHANGES_REQUESTED** [review]

File: src/components/ArrowControl/ArrowControl.js (hunk 1 above)
Comment (klubiniecki-qantas) [reply src/components/ArrowControl/ArrowControl.js:11]: Will do!

File: src/components/ArrowControl/ArrowControl.js (hunk 2 above)
Comment (klubiniecki-qantas) [reply src/components/ArrowControl/ArrowControl.js:28]: 👍

File: src/components/ArrowControl/ArrowControl.js (hunk 1 above)
Comment (apalski) [reply src/components/ArrowControl/ArrowControl.js:11]: Maybe consider the aria-controls [(https://www.w3.org/TR/wai-aria-practices-1.2/#button)]

**Review by apalski — APPROVED** [review]
Looks great 👍  just a couple of comments

**Review by KieraDOG — APPROVED** [review]

**Pull request merged**